lib/slide_hub/convert_procedure.rb

```diff
@@ -110,14 +110,16 @@ def upload_files
     def update_database
       if ENV.fetch('OSS_MULTI_TENANT') { false }
         tenants = Tenant.pluck(:name)
+        logger.info("all tenants are #{tenants.to_s}")
         tenants.each do |tenant|
           Apartment::Tenant.switch(tenant) do
+            logger.info("tenant is #{tenant}")
             update_slide
           end
         end
-        Apartment::Tenant.reset do
-          update_slide
-        end
+        logger.info('then check master database...')
+        Apartment::Tenant.reset
+        update_slide
```

Comment:
__[RuboCop]__ <sub>posted by Sider</sub>

Style/IdenticalConditionalBranches: Move `update_slide` out of the conditional.